src/views/MetocDataView.vue

```diff
@@ -38,11 +38,6 @@
             :items="dataSources"
             @input="onSelectDataSource"
           />
-          <WMSInfoPanel
```

Comment:
`WMSInfoPanel` is not in any other part of the code so we can delete it

Reply:
Since this is a branch from the WebOC where it will be used, I was not sure if I should remove it indeed